src/dimensions.ts

```diff
@@ -651,19 +682,28 @@ Dimensions.prototype.getInputsVSize = function(this: IBaseDimensions) {
     txP2shP2wshInputSize,
     txP2wshInputSize,
     txP2trKeypathInputSize,
+    txP2shP2pkInputSize,
   } = VirtualSizes;
 
   const {
     nP2shInputs,
     nP2shP2wshInputs,
     nP2wshInputs,
     nP2trKeypathInputs,
+    nP2shP2pkInputs,
   } = this;
 
-  return nP2shInputs * txP2shInputSize +
+  const size = nP2shInputs * txP2shInputSize +
     nP2shP2wshInputs * txP2shP2wshInputSize +
     nP2wshInputs * txP2wshInputSize +
-    nP2trKeypathInputs * txP2trKeypathInputSize;
+    nP2trKeypathInputs * txP2trKeypathInputSize +
+    nP2shP2pkInputs * txP2shP2pkInputSize;
+
+  if (Number.isNaN(size)) {
```

Comment:
Mostly curious - when would `size` be `NaN`? Would its components ever not be numbers?